src/testnet_verifier/testnet_verifier.star

```diff
@@ -19,7 +19,7 @@ def run_synchronous_testnet_verification(params, el_client_contexts, cl_client_c
 	add_service(SERVICE_ID, config)
 
 	command = get_cmd(params, el_client_contexts, cl_client_contexts, True)
-	exec(SERVICE_ID, command)
+	exec(struct(service_id=SERVICE_ID, command=command))
```

Comment:
can we add an assertion for exit code being 0?

the previous one would fail at execution time if exit code wasn't 0